github/event_types.go

```diff
@@ -1248,3 +1248,63 @@ type WorkflowRunEvent struct {
 	Sender       *User         `json:"sender,omitempty"`
 	Installation *Installation `json:"installation,omitempty"`
 }
+
+type SecurityAdvisory struct {
```

Comment:
Please add godoc-style comments to each of the new structs you are adding to state their purpose.